databases/week2/school_database.sql

```diff
@@ -0,0 +1,59 @@
+-- DROP DATABASE IF EXISTS `lesson2_homework_school_database`;
+-- SHOW DATABASES;
+
+-- Create a new database containing the following tables:
+CREATE DATABASE IF NOT EXISTS `lesson2_homework_school_database`;
+
+USE `lesson2_homework_school_database`;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE IF NOT EXISTS `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begins` DATETIME NOT NULL,
+  `ends` DATETIME NOT NULL,
```

Comment:
Data type is `date` not `datetime` here. There is a slight dif between those. FYI

Reply:
Understood the difference